### DISCUSSION TITLE
My edu email not display in apply github education pack form

### DISCUSSION THREAD

**Discussion by xuan-giang-it** [community/community#35288] [GitHub Education]
Hi everyone, I don't see my email in form apply edu pack. I verified my email but it not display, please help me
Thank you very much!
![image](https://user-images.githubusercontent.com/115130074/194209610-30f5252d-22f5-491c-941e-9651e04626c3.png)
![image](https://user-images.githubusercontent.com/115130074/194209706-e2bdbf60-f105-4fc2-a4f8-50d4ad80c8a7.png)

**Comment by samus-aran** [comment]
Hi @xuan-giang-it 👋 
I suspect that the Global Campus application page just needs some time to re-gather your account's verified emails if you've only recently added it. Have you been able to refresh that page since? If so are you still receiving that error?

**Reply by xuan-giang-it** [reply]
Hi @samus-aran ,
Today I entry form apply education pack but it still not display my email. 
Please help me, thank you very much
![ảnh](https://user-images.githubusercontent.com/115130074/194690247-ff3a93d2-a2d0-40f8-9f5b-8374b910f745.png)

**Reply by Laendrun** [reply]
I have the same issue, I told myself I'd wait some time before checking again, but after 16 hours, my edu email stil doesn't show in the application form.

**Comment by samus-aran** [comment]
Hi @xuan-giang-it & @Laendrun. 

Are you able to delete your browser cache and revisit the application form page again. Hopefully this will pick up your verified emails.

Let us know how you get on.

**Reply by Laendrun** [reply]
Just tried that, nothing better, I also saw that all my email are listed up to my primary email :
- simon.aeby@laendrun.ch
- email2@laendrun.ch - primary
- school@email.ch

in the application form I have the first two adresses, maybe this has an impact (not sure how it could impact that but why not)

**Reply by xuan-giang** [reply]
I tried again but nothing better.

**Comment by samus-aran** [comment]
Thanks @xuan-giang-it & @Laendrun !

We've been able to replicate this on our end and have raised an issue with the team. We'll comment when we have any further updates. 😃

**Reply by Laendrun** [reply]
Thank you for your quick reaction and fast replies !

**Comment by samus-aran** [comment]
Hey @Laendrun & @xuan-giang-it 👋 
The team has pushed up a fix. Can you re try the application page again and confirm that you can now see your verified emails 👍

**Reply by Laendrun** [reply]
All good, I can see my school email now, though it says I have to check with my school admin to have the correct benefits applied to my account, but the school email is now displayed, thank you for your fix !

**Comment by RafaelGPL** [comment]
Hi, I am having this issue or a variation of it: when I go to the pack application page, by default the "student" radio button is selected and my academic email shows up in the list of selectable email addresses but as soon as I select "teacher" (which is what I need) it removes only that email address from the list (showing my two other email addresses associated with my account).

**Comment by 1zoecamp** [comment]
@samus-aran hello! I do have the same problem, and when I try to add again my educational email I get the notification that my email is already in use... I also tried to log in with that email, which was successfull but stays the same when I tried to get my educational package

**Comment by Harsh-0986** [comment]
This issue has started showing up again, can the team please look up to it.
